### ISSUE TITLE
Reject proposals

### ISSUE THREAD

**Issue by raucao** [67P/kredits-contracts-legacy#12]
We need a way to remove proposals that don't make sense. I image that, analogous to upvoting/accepting, this could be done by downvoting and deleting upon reaching the desired quorum (just 2 to begin with for example).

**Comment by bumi** [comment]
yep. makes sense. I would not delete the proposals for transparency reasons but just flag them as rejected.
